recipes/knot/meta.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: knot
+  version: "1.0.0"
+
+source:
+  fn: KNOT-1.0.0.tgz
+  url: http://mathbio.nimr.mrc.ac.uk/download/KNOT/KNOT-1.0.0.tgz
+  md5: 2556876109bae506fe171a45902db0b6
+
+build:
+  number: 1
+
+requirements:
+  build:
+    - system
```

Comment:
Can you comment on this dependency? I think if there are no real dependencies, you can also just skip the whole section.